meshroom/ui/qml/GraphEditor/ScriptEditor.qml

```diff
@@ -182,31 +210,50 @@ Item {
             }
         }
 
-        RowLayout {
-            Label {
-                text: "Input"
-                font.bold: true
-                horizontalAlignment: Text.AlignHCenter
-                Layout.fillWidth: true
-            }
+        MSplitView {
```

Comment:
This needs to be discussed but I wonder if it wouldn't be better to have the split view be horizontal instead of vertical... As such, it's is pretty much unusable unless we expand the Script Editor's tab, which may not be ideal.

Reply:
Thanks for your feedback @cbentejac, I do agree with how the widgets are placed in Meshroom at the moment, the best layout is a RowLayout. 
Have reverted back in terms of Layout and I guess we can keep it till the time we have a better location/placement of Script Editor.